### PR TITLE
Services update

### PR DESCRIPTION
**Pull request by rafsalas19**
- Slight modification to install scripts to remove unused variable and slight change to stream line Managed Prometheus launch.
- Services Configuration now calls install to ease burden of users
- Service files modified to launch managed Prometheus and remove Geneva
- Documentation update for services
- Process check that returns whether or not services and containers launched correctly
- Small bug fixes

### CODE REVIEW

File: linux_service/start_moneo_services.sh [hunk 1]
```diff
@@ -1,6 +1,47 @@
 #!/bin/bash
-PUBLISHER=$1
-MONEO_PATH=$2
+PUBLISHER=$3
```
Comment (moes1) [inline linux_service/start_moneo_services.sh:2]: This name seems a bit confusing for this variable if it is supposed to be true or false? Could it be called WITH_PUBLISHER or something?

Comment (rafsalas19) [reply linux_service/start_moneo_services.sh]: ok I addressed. Please have another look.

**Review by RyoYang — APPROVED** [review]

**Review by RyoYang — APPROVED** [review]

**Review by moes1 — APPROVED** [review]

**Comment by PPPW** [comment]
Hi @rafsalas19, we found Moneo was not working for us and traced down to this breaking change. We are using Geneva, which is not supported anymore. It's still supported if we run `moneo.py` though. I wonder:

- Any chance we will still support Geneva in "linux_service"? It won't be very hard to support it here, mostly, we just need to call `src/worker/start_geneva.sh`. But I think your team must have some design concerns behind this breaking change.
- Will we remove support for Geneva even in the `moneo.py` in the future?
- Can we get some notification for breaking changes in the future?

Thank you!

**Comment by RyoYang** [comment]
Hey @PPPW, I just noticed your message, and I apologize for the changes that caused a break. I'm not sure if Rafael has responded to you yet, so I'll address your question here in case Rafael has ignored the message.

1. Yes, we are able to roll back the previous change and support Geneva publisher as a Linux service. We will do this later. In the meantime, you can use the previous version and patch some bug-fix commits as a workaround.

2. Indeed, managed Prometheus will be the next step for Moneo, and internally, we have deprecated the Geneva solution. However, Moneo will still maintain Geneva support but may not actively develop it further unless your team has a feature request.

3. I apologize once again for making these changes without any notification. I think we should use release notes on the 'issues' page to include planned features for this release. This way, developers/users will be able to track the updates to Moneo.

Hey @rafsalas19, not sure if you have more suggestions on this.

**Comment by PPPW** [comment]
Hi @RyoYang, thank you for the reply! For the Linux service, we are using "moneo.py" with a hostfile that contains a node itself as a workaround, so it's working fine so far. As we discussed with our team, we think it makes sense for your team to stay with your plans and designs, rather than keep supporting Geneva just for us. But it'll be great if you could notify our team when you are going to remove Geneva support in "moneo.py", so that we have some time to look into managed Prometheus or other options. Thanks!

**Pull request merged**